livre/eq/settings/FrameSettings.cpp

```diff
@@ -44,24 +45,33 @@ void FrameSettings::reset()
 void FrameSettings::serialize( co::DataOStream& os, const uint64_t dirtyBits )
 {
     co::Serializable::serialize( os, dirtyBits );
-    os << currentViewId_ << frameNumber_ << screenShot_
+    os << currentViewId_ << frameNumber_ << frameRange_ << screenShot_
        << recording_ << statistics_ << help_ << grabFrame_;
 }
 
 void FrameSettings::deserialize( co::DataIStream& is, const uint64_t dirtyBits )
 {
     co::Serializable::deserialize( is, dirtyBits );
-    is >> currentViewId_ >> frameNumber_ >> screenShot_
+    is >> currentViewId_ >> frameNumber_ >> frameRange_ >> screenShot_
        >> recording_ >> statistics_ >> help_ >> grabFrame_;
 }
 
 void FrameSettings::setFrameNumber( uint32_t frame )
 {
-    if( frameNumber_ == frame )
-        return;
+    if( frameNumber_ != frame )
+    {
+        frameNumber_ = frame;
+        setDirty( DIRTY_ALL );
+    }
+}
 
-    frameNumber_ = frame;
-    setDirty( DIRTY_ALL );
+void FrameSettings::setFrameRange( const Vector2ui& frameRange )
+{
+    if( frameRange_ != frameRange )
+    {
+        frameRange_ = frameRange;
+        setDirty( DIRTY_ALL );
+    }
```

Comment:
No, just consistency. Both ways were present (setGrabFrame) and I chose this one to unify them all.
Anyway, this whole commit won't be merged for now, so don't review. We are trying to find a better way to do the same.